lib/livebook_cli/server.ex

```diff
@@ -15,12 +15,20 @@ defmodule LivebookCLI.Server do
   @impl true
   def usage() do
     """
-    Usage: livebook server [url] [options]
+    Usage: livebook server [open-command] [--options]
```

Comment:
@josevalim what about `livebook server [options] [open-command]`?

I think it's more common to list options first. Also `--options` looks like it was an actual option.

Reply:
I will change!